modules/tinymce/src/themes/silver/main/ts/api/Events.ts

```diff
@@ -39,6 +39,30 @@ const fireToggleToolbarDrawer = (editor: Editor, state: boolean): void => {
   editor.dispatch('ToggleToolbarDrawer', { state });
 };
 
+const fireStylesTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('StylesTextUpdate', data);
+};
+
+const fireAlignTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('AlignTextUpdate', data);
+};
+
+const fireFontSizeTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('FontSizeTextUpdate', data);
+};
+
+const fireFontSizeInputTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('FontSizeInputTextUpdate', data);
+};
+
+const fireBlocksTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('BlocksTextUpdate', data);
+};
+
+const fireFontFamilyTextUpdate = (editor: Editor, data: { value: string }): void => {
+  editor.dispatch('FontFamilyTextUpdate', data);
+};
```

Comment:
Please make a docs ticket to describe the new events. They might prove useful to integrators.

@spocke are you happy with these event names?

Reply:
Good catch, I've created [DOC-2202](https://ephocks.atlassian.net/browse/DOC-2202).